app/models/user.rb

```diff
@@ -31,7 +31,7 @@
 
 class User < ApplicationRecord
   belongs_to :organization, optional: proc { |u| u.super_admin? }
-
+  has_many :feedback_messages, dependent: :destroy
```

Comment:
I wonder if they should be `dependent: :destroy`? I think even if the user is removed we may want a history of the various pieces of feedback? Although I don't know if we currently have a way to delete users so this might be moot.